bin/ODiffBin.re

```diff
@@ -10,15 +26,55 @@ let diffPath =
 let base =
   Arg.(
     value
-    & pos(0, file, "")
-    & info([], ~docv="BASE", ~doc="Path to base image")
+    & pos(0, underscore_or(non_dir_file), "_")
```

Comment:
You are now able to add either a file or an `_` (underscore) to accept a raw image buffer.
I explicitly added the `non_dir_file` converter, so directories are not a valid input.